docs/concepts/executor/hub/use-hub-executor.md

```diff
@@ -150,13 +150,13 @@ You can find the Executor by running `docker images`. You can also indicate whic
 jina hub pull jinaai+docker://jina-ai/DummyExecutor:v1.0.0
 ```
 
-### Platform awareness of hub images 
+### Platform awareness of Hub images 
 
 ````{admonition} Hint
 :class: hint
 As of January 10, 2023 `jina hub pull` is platform aware. It will automatically select docker images base on your native CPU architecture (if available).
```

Comment:
```suggestion
As of January 10, 2023 `jina hub pull` is platform aware. It will automatically select Docker images based on your native CPU architecture (if available).
```